src/gui/mainwindow.cpp

```diff
@@ -1239,7 +1238,21 @@ void MainWindow::optionsSaved()
     loadPreferences();
 }
 
-// Load program preferences
+void MainWindow::showStatusBar(bool show)
+{
+    if (show && !m_statusBar) {
+        // Create status bar
+        m_statusBar = new StatusBar;
+        connect(m_statusBar, &StatusBar::connectionButtonClicked, this, &MainWindow::showConnectionSettings);
```

Comment:
Microsoft compiler is not standard compliant then, as far as I understand, and you need to fix the very same error at other places.